include/verilated.cpp

```diff
@@ -3000,8 +3000,8 @@ VerilatedScope::~VerilatedScope() {
 }
 
 void VerilatedScope::configure(VerilatedSyms* symsp, const char* prefixp, const char* suffixp,
-                               const char* identifier, int8_t timeunit,
-                               const Type& type) VL_MT_UNSAFE {
+                               const char* identifier, int8_t timeunit, const Type& type,
+                               const char* defName) VL_MT_UNSAFE {
```

Comment:
There's an error with t_protect_ids where it calls this from cpp, should I add a default value, or just put "" in the test?